rules/falco_rules.yaml

```diff
@@ -2007,7 +2007,7 @@
 # when we lose events and lose track of state.
 
 - macro: container_entrypoint
-  condition: (not proc.pname exists or proc.pname in (runc:[0:PARENT], runc:[1:CHILD], runc, docker-runc, exe, docker-runc-cur))
+  condition: (not proc.pname exists or proc.pname in (runc:[0:PARENT], runc:[1:CHILD], runc, docker-runc, exe, docker-runc-cur, containerd-shim-runc-v2, systemd))
```

Comment:
Include `systemd` for robustness or fail safe as in odd edge cases this could happen with the current logic.

Reply:
what about `podman`?

Reply:
Absolutely, we need to check for each supported container engine. Was hoping it's gonna be a team effort. For this first PR it's only definitely valid for containerd, rest we do in as many follow up PRs as we need.